packages/core/src/util/renderer.ts

```diff
@@ -702,15 +709,16 @@ export const mapStateToLayoutProps = (
   ownProps: OwnPropsOfLayout
 ): LayoutProps => {
   const rootData = getData(state);
+  const ajv = getOrCreateAjv(state);
```

Comment:
This should be `get` and all the functions should just use this variable.

Reply:
Fixed